healthtools/tests/test_scrapers.py

```diff
@@ -68,48 +43,46 @@ def test_it_scrapes_nhif_outpatient_cs_page(self):
     def test_doctors_scraper_uploads_to_elasticsearch(self):
         with open(self.TEST_DIR + "dummy_files/doctors.json", "r") as my_file:
             data = my_file.read()
-            response = self.doctors_scraper.elasticsearch_index(json.loads(data))
-            self.assertEqual(response["items"][0]["index"]["_shards"]["successful"], 1)
+        response = self.doctors_scraper.elasticsearch_index(json.loads(data))
+        self.assertEqual(response["items"][0]["index"]["created"], True)
 
     def test_foreign_doctors_scraper_uploads_to_elasticsearch(self):
         with open(self.TEST_DIR + "dummy_files/foreign_doctors.json", "r") as my_file:
             data = my_file.read()
-            response = self.foreign_doctors_scraper.elasticsearch_index(json.loads(data))
-            self.assertEqual(response["items"][0]["index"]["_shards"]["successful"], 1)
+        response = self.foreign_doctors_scraper.elasticsearch_index(json.loads(data))
+        self.assertEqual(response["items"][0]["index"]["created"], True)
 
     def test_clinical_officers_scraper_uploads_to_elasticsearch(self):
         with open(self.TEST_DIR + "dummy_files/clinical_officers.json", "r") as my_file:
             data = my_file.read()
-            response = self.clinical_officers_scraper.elasticsearch_index(json.loads(data))
-            self.assertEqual(response["items"][0]["index"]["_shards"]["successful"], 1)
+        response = self.clinical_officers_scraper.elasticsearch_index(json.loads(data))
+        self.assertEqual(response["items"][0]["index"]["created"], True)
 
     def test_health_facilities_scraper_uploads_to_elasticsearch(self):
         with open(self.TEST_DIR + "dummy_files/health_facilities.json", "r") as my_file:
             data = my_file.read()
-            response = self.health_facilities_scraper.elasticsearch_index(json.loads(data))
-            self.assertEqual(response["items"][0]["index"]["_shards"]["successful"], 1)
+        response = self.health_facilities_scraper.elasticsearch_index(json.loads(data))
+        self.assertEqual(response["items"][0]["index"]["created"], True)
 
     def test_nhif_inpatient_scraper_uploads_to_elasticsearch(self):
         with open(self.TEST_DIR + "dummy_files/nhif_inpatient.json", "r") as my_file:
             data = my_file.read()
-            response = self.nhif_inpatient_scraper.elasticsearch_index(json.loads(data))
-            self.assertEqual(response["items"][0]["index"]["_shards"]["successful"], 1)
+        response = self.nhif_inpatient_scraper.elasticsearch_index(json.loads(data))
+        self.assertEqual(response["items"][0]["index"]["created"], True)
```

Comment:
That is it. I spent quite some time trying to refactor this since running the elastic search locally and on aws seemed to return different results but checking the number should work better 👍